client/CVideoHandler.cpp

```diff
@@ -185,48 +185,62 @@ bool CVideoPlayer::open(std::string fname, bool loop, bool useOverlay, bool scal
 		destRect.h = pos.h;
 	}
 
-	if (texture == nullptr && dest == nullptr)
+	if(texture == nullptr && dest == nullptr)
 		return false;
 
-	if (texture)
+	if(texture)
 	{ // Convert the image into YUV format that SDL uses
 		sws = sws_getContext(codecContext->width, codecContext->height, codecContext->pix_fmt,
-							 pos.w, pos.h,
-							 AV_PIX_FMT_YUV420P,
-							 SWS_BICUBIC, nullptr, nullptr, nullptr);
+		pos.w, pos.h,
+		AV_PIX_FMT_YUV420P,
+		SWS_BICUBIC, nullptr, nullptr, nullptr);

```

Comment:
This is not really intended and there must be single tab indention. Probably I should really leave all lambdas completely broken for now because  "indent_continue = -1" confuse everyone.